apps/src/panels/PanelsView.tsx

```diff
@@ -159,6 +160,18 @@ const PanelsView: React.FunctionComponent<PanelsProps> = ({
   }, [currentPanelIndex, setTypingDone]);
 
   const panel = panels[currentPanelIndex];
+  const showTyping =
+    panel.typing || queryParams('panels-show-typing') === 'true';
```

Comment:
This new ordering can leave `panel` as `undefined`.  Tiny fix-forward prepped in https://github.com/code-dot-org/code-dot-org/pull/69672.